tools/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 echo "Setting up coverage logging";
-mkdir coverage
+mkdir -p coverage
```

Comment:
add -p to mkdir to avoid errors on call to mkdir 
`-p, --parents     no error if existing, make parent directories as needed`